examples/grpc/src/main.rs

```diff
@@ -109,88 +107,81 @@ impl Greeter for MyGreeter {
     }
 }
 
-mod connector {
-    use hyper::Uri;
-    use hyper_util::client::legacy::connect::Connected;
-    use hyper_util::rt::TokioIo;
+mod incoming {
```

Comment:
We could avoid this by implementing in turmoil, but we'd have to take a tonic dependency and managing version upgrades will be difficult.

Reply:
I think this is fine for something that users can copy into their own code pretty easily and so we dont need to pin turmoil to tonic.